hack/update-codegen.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SRC="github.com"
+GOPKG="$SRC/mikkeloscar/kube-aws-iam-controller"
+CUSTOM_RESOURCE_NAME="amazonaws.com"
```

Comment:
Again, I suggest reconsidering use of this name for which we're not the trademark holder.